custom/icds/integrations/phi.py

```diff
@@ -13,6 +13,8 @@
 
 
 def send_request(request_type, data):
+    if isinstance(data, dict):
+        data = json.dumps(data)
```

Comment:
The content type should always be JSON.
This just handles the case if the data passed is not already a JSON string i.e a dict. Currently the repeater calls this and data is a JSON but while i was testing it, realised that this would not be the case always.